db/seeds/muscle_group_descriptions.rb

```diff
@@ -0,0 +1,22 @@
+  "The deltoid as a whole is composed of three small muscle heads, therefore is a relatively small muscle group. Basically, there are two types of exercises for the deltoids; presses and raises. These used in variations will train all three heads effectively. It is proven that low reps are best for all pressing movements and high reps are most effective for all raising movements."
+
+  BACK =
+  "The back is divided into three generalizations, the thickness, width and detail of the lower back. Of course, there are a variety of exercises available which target each area specifically. The two main movements or type of movements which stimulate the back muscles are the range of movement of the shoulders being pulled back or down. In addition, for the lower back, since it's function is that of a stabilizer, you must actually bend at the waist to stimulate its growth. The trapezius muscles which form the basis of the upper back work in opposition to the exercises of the back. However, the trapezius are trained as a secondary muscle for many exercises of other body parts. The exercises which are beneficial are shrugging movements."
+
+  LEGS =
+  "The thighs in general are tough mentally and physically to train, when done so correctly. It takes dedication and commitment to build balanced thighs. For the quadriceps as well as the hamstrings, it is very important to use a full range of motion in all exercises. While training quadriceps, it is important to incorporate basic foundation movements with isolating movements. While training the thighs you will also target your buttocks as a secondary muscle. This is especially effective for squatting and lunging exercises. In addition, hamstrings are a muscle which must be fully stretched at the extension phase of the movement while trained. High reps are most effective for isolation movements, where low rep are for compound exercises such as squats can be a good muscle builder. You must also take care to stretch and warmup prior to any exercise, especially the knees and other susceptible joints."
+
+  BUTTOCKS =
+  "Basically, most leg exercise will target the buttock directly or indirectly. Common movements such as squats and lunges have a profound effect on the strength and development of this region."
+
+  CHEST =
+  "The chest is a muscle which must be trained for all of its areas to develop completely. Usually for males, a full and wide chest is the goal. It is very important as with most large muscle groups to build a foundation of the muscle before concerning yourself with details. This foundation phase will be the building blocks to further detailed work. With the pectoral muscles, it is significant to follow a plan which will consist of a few basic movements along with some isolation movements. Proper form, contraction and stretching will boost your gains and help prevent injuries. There is a wide range of exercise to properly stimulate all areas of the chest muscles for men and women."
```

Comment:
Line is too long. [690/80]<br>Indent the first line of the right-hand-side of a multi-line assignment.<br>Freeze mutable objects assigned to constants.